NodeJS/README.md

```diff
@@ -88,6 +88,34 @@ The AUXCodeClient retrieves sub account aux codes, effective aux code given sub
     // get subaccount effective aux code via sub account app id
     let appIdAuxCodes = await auxCodeClient.getAUXCodesForEffectiveAppId();
 
+## AddressBookClient
+
+The AddressBookClient could be used to get sub-account Address Book with all active contacts.
+With search endpoint contacts could be fetched by multiple optional params: type (one or many of Agent/Supervisor/Outbound/ACO/VDN separated by comma); partial name/destination; full virtualExtension.
+The search request could contain pagination (pageSize; page number) and sorting (by name/type/virtualExtension/destination; direction ASC/DESC) params.
+
+    // create an instance of AddressBookClient
+    let addressBookClient = createAddressBookClient(endpoint, apiKey)
```

Comment:
```suggestion
    const addressBookClient = createAddressBookClient(endpoint, apiKey)
```

And you need to put code [in code blocks](https://commonmark.org/help/tutorial/09-code.html)

Reply:
also, no need to indent